selection_sort/selection.cpp

```diff
@@ -0,0 +1,38 @@
+#include<bits/stdc++.h>
```

Comment:
Use a function for the code and use the main function for calling the sort function!

Check out other codes for reference in the repo!
small changes and you are good to go! :tada: